pkg/k8shandler/reconciler.go

```diff
@@ -74,13 +73,13 @@ func ReconcileForClusterLogForwarder(forwarder *logging.ClusterLogForwarder, req
 
 	clusterLogging := clusterLoggingRequest.getClusterLogging()
 	if clusterLogging == nil {
-		logger.Debug("No clusterlogging object, cannot reconcile")
+		log.Info("No clusterlogging object, cannot reconcile")
```

Comment:
We should reconsider the value of moving this to Info since it will fill the logs with info of questionable value.  @alanconway ?

Reply:
I deleted these kind of logs in EO because we determined they were useless.

Reply:
deleted